app/serializers/event_serializer.rb

```diff
@@ -1,3 +1,6 @@
+require 'benchmark'
```

Comment:
Why do you require `benchmark`?

Reply:
leftover from testing will get rid of it